app/uk/gov/hmrc/personaldetailsvalidation/views/template/personal_details.scala.html

```diff
@@ -28,51 +28,56 @@
   viewConfig: ViewConfig
 )
 
-@main_template(title = messages("personal-details.title")) {
+@main_template(title = error_prefix(pageForm) + messages("personal-details.title") + " - GOV.UK") {
```

Comment:
Should the '- GOV.UK' suffix be appended to the title in the main template, so all pages will get it automatically?